posthog/clickhouse/property_groups.py

```diff
@@ -109,6 +109,29 @@ def get_alter_create_statements(
 
         yield f"{prefix} " + ", ".join(commands)
 
+    def get_alter_modify_statements(
+        self,
+        table: TableName,
+        source_column: PropertySourceColumnName,
+        group_name: PropertyGroupName,
+        cluster: str | None = None,
+    ) -> Iterable[str]:
+        """
+        Returns an iterable of ALTER TABLE statements that can be used to modify the property group
+        using MODIFY COLUMN statements.
+        """
```

Comment:
Does this invalidate the existing materializations already run? Or does this leave the existing data that has already been written? If it doesn't invalidate the existing data it might be good to clarify that this should only be used to remove values from the map, not add them - but I'm not sure how this actually works under the hood